internal/operator/subscription.go

```diff
@@ -99,16 +99,21 @@ func (c operatorClient) ListPackageManifests(ctx context.Context, list *pkgserve
 
 	if len(filter) > 0 {
 		var notFound []string
+		wasFound := false
 		for _, f := range filter {
 			for _, pkgm := range tmppkgmlist.Items {
 				if pkgm.Name == f {
 					list.Items = append(list.Items, pkgm)
-					notFound = append(notFound, f)
+					wasFound = true
```

Comment:
I would suggest using just a boolean and not storing anything in a list. Something like this?:
```
if len(filter) > 0 {
		notFound := false
		for _, f := range filter {
			for _, pkgm := range tmppkgmlist.Items {
				if pkgm.Name == f {
					list.Items = append(list.Items, pkgm)
				}
				notFound = true
			}
		}
		if notFound {
			return fmt.Errorf("Package %v not found in catalog source", notFound)
		}
}

Reply:
Sorry instead of `return fmt.Errorf("Package %t not found in catalog source", notFound)` try this: 
`return fmt.Errorf("package %v not found in catalog source", filter)`

Reply:
@trgeiger , @yashoza19 suggestions seems right. Let us know when it's ready for testing.

Reply:
@trgeiger Sorry, I was just suggesting a single package not found. In the case of a list of filter packages, we would require them to be stored in a NotFound list. Sorry for missing that.

Reply:
No worries, I can fix this up real quick to handle multiple missing packages.

Reply:
@yashoza19 it now should handle multiple missing packages, or a single missing package from the filter